src/content/1.7/faq/stock.md

```diff
@@ -36,51 +36,63 @@ OutOfStockType::OUT_OF_STOCK_DEFAULT = 2; // Use default configuration
+| ------------- | ---------------- | ----------------- | ----------------- | --------------- |
+| `Great Product` is created in the BO with a virtual quantity of 50 | `50` | `50` | `0` | |
 | A customer decides to add 3 `Great Product` into their cart | `50` | `50` | `0` | For now the product is only in a cart, nothing is reserved, anyone can still buy one of the 50 remaining products. |
-| The customer goes through the whole checkout process and chooses to pay by check. | `47` | `50` | `3` | The order has been created, but the payment is not accepted yet. |
-| A few days later, the merchant receives the check, so they change the Order status to `Payment accepted` | `47` | `50` | `3` | The product is now paid, but it still has not left the shelves. |
+| The customer goes through the whole checkout process and chooses to pay by check. | `47` | `50` | `3` | The order has been created, but the payment is not accepted yet. Virtual quantity and reserved quantity are updated. |
+| A few days later, the merchant receives the check, so they change the Order status to `Payment accepted` | `47` | `50` | `3` | The product is now paid, but the package has still not been prepared. |
 | The package is prepared and delivered to a Carrier, the order status is switched to `Delivered` or `Shipped` | `47` | `47` | `0` | The product is no longer reserved, it has actually been sent to the customer. So reserved and physical quantities can be decreased. |
 
 **Q:** Where can I see and modify my Stock?
 
-**A:** Although you can change the quantity directly on your product page, we strongly recommend using the `Catalog > Stock` section in your BO because it is based on `StockMovement` which is safer than a direct edition ([read below](#stockmovement)).
+**A:** Although you can change the quantity directly on your product page, we strongly recommend using the `Catalog > Stock` section in your BO because it is based on `StockMovement` which is safer than a direct edition ([read below why](#stockmovement)).
 
-**Q:** Can I edit the physical or reserved quantity myself? or using an ERP?
+**Q:** Can I edit the physical or reserved quantity myself or using an ERP?
 
-**A:** NO, these two fields should be considered as read only, they made the display of stock data faster. They are automatically recomputed when the stock is synchronized:
+**A:** Avoid doing this! These two fields should be considered read-only and exist for performance reasons. They are automatically recomputed every time the stock is synchronized:
 
-- **reserved_quantity**: is the sum of each quantity in pending orders `reserved_quantity_per_order = SUM(order_detail.product_quantity - order_detail.product_quantity_refunded)`
-- **physical_quantity**: once the reserved quantity is updated based on Orders, the physical one is computed like this `physical_quantity = available_quantity + reserved_quantity`
+- **reserved_quantity** is the sum of each quantity in pending orders:
```

Comment:
You need two spaces at the end of the line for the line break to be taken into account
```suggestion
- **reserved_quantity** is the sum of each quantity in pending orders:  
```